core/objectUtils.js

```diff
@@ -57,6 +55,10 @@ export const getAuthGroups = R.propOr([], keys.authGroups)
 
 export const isTemporary = isKeyTrue(keys.temporary)
 
+// ====== CHECK
+const isBlank = (value) => value === null || value === undefined || R.isEmpty(value) || StringUtils.isBlank(value)
```

Comment:
don't we have A.isEmpty ?

Reply:
uhm, right, I don't know why it was there... old piece of code